.gitignore

```diff
@@ -1 +1,16 @@
-/Build
\ No newline at end of file
+/Build
+.cxx
+*.iml
+*.so
+local.properties
+.gradle
+.idea
+Library/Dependencies/v8/include
+.gradle
+.DS_Store
+build/
+captures/
+.externalNativeBuild
+*.apk
+*.ap_
+*.log
```

Comment:
I don't think the Android ignore list should be at the root. At the minimum, put this in the Android specific folder.

Reply:
That means the gitignore have to be updated for every Android Studio project (testapp + integration for now). Are you ok with that?

Reply:
There has to be a better way to do this. Ignoring all .cxx, .log, .so, etc. seems a bit extreme.